src/ffi/datetime.rs

```diff
@@ -102,6 +106,7 @@ const _PyDateTime_DATETIME_DATASIZE: usize = 10;
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
 pub struct PyDateTime_Date {
+    /// Structure representing a `datetime.date`
```

Comment:
Unlike python, rust wants the doc strings outside the struct (The current would document `ob_base`)

Reply:
Yeah, I noticed this about halfway through, then went back to fix them. Must have missed at least this one.